docs/learn/smart-contract-developers/create-lsp7-token.md

```diff
@@ -9,48 +9,65 @@ import TabItem from '@theme/TabItem';
 
 # Create an LSP7 Token
 
+This guide will walk you through the process of creating and deploying a custom [LSP7 Digital Asset](../../standards/tokens/LSP7-Digital-Asset.md) and pre-mint a certain amount of tokens to a the token owner. To build a smart contract using LSPs, you can **inherit functionality** from modular and standardized presets in the [`@lukso/lsp-smart-contracts`](../../tools/lsp-smart-contracts/getting-started.md) library. To learn more about the contract standards itself, please refer to the [Contracts section](../../contracts/introduction.md) of our documentation.
+
 :::tip
 
-You can learn about the project setup by checking the [Getting Started](./getting-started.md) page.
+You can learn about the project setup and Hardhat workflow by checking the [Getting Started](./getting-started.md) section.
 
 :::
 
-:::note
+## Create the Token
+
+:::info
 
-For the following guide, we are using `hardhat v2.19.1`, `ethers v6` and `@lukso/lsp-smart-contracts`
-.
+For instructions on deploying a contract using the Universal Profile Browser Extension, please refer to the [Deploy Contracts Guide](../dapp-developer/deploy-contracts.md) for dApp developers.
 
 :::
 
-## Create a custom LSP7 Token
-
-:::info
+:::tip Code repository
 
-This guide will walk you through the process of deploying a contract in the context of Hardhat. For instructions on deploying a contract within a dApp using the UniversalProfile Browser extension, please refer to the [Deploy Contracts section](../dapp-developer/deploy-contracts.md) in the dApp developers section.
+You can find all the contracts and scripts of the guide within our [`lukso-playground`](https://github.com/lukso-network/lukso-playground) repository.
 
 :::
 
-In this guide you will create a custom [LSP7 Digital Asset](../../standards/tokens/LSP7-Digital-Asset.md) and pre-mint a certain amount of tokens to a specific address. To build your smart contract you will use the following LSP7 preset and extension:
+For our sample deployment of the LSP7 token, will will use the following presets:
 
 - [`LSP7Mintable`](../../contracts/contracts/LSP7DigitalAsset/presets/LSP7Mintable.md): allow creating new assets on the smart contract.
 - [`LSP7Burnable`](../../contracts/contracts/LSP7DigitalAsset/extensions/LSP7Burnable.md): allow tokens to be removed from the supply.
 
-You can modify the `mint()` function to adjust the amount of initially minted tokens and their receiver as described in the [LSP7 Mintable Documentation](../../contracts/contracts/LSP8IdentifiableDigitalAsset/presets/LSP8Mintable.md#mint).
+You can then import them within your Solidity contract file:
 
 ```solidity title="contracts/MyCustomToken.sol"
 // SPDX-License-Identifier: Apache-2.0
-pragma solidity ^0.8.4;
+pragma solidity ^0.8.9;
```

Comment:
fine with me but any particular reason we bumped to `^0.8.9;`?

Reply:
Just to be in sync with the playground updates